lambdas/cloudflare_rematch/tsconfig.json

```diff
@@ -0,0 +1,105 @@
+{
```

Comment:
It feels like there's a lot of commented-out code in this file -- maybe we should just keep the link to whatever the reference is in a comment and delete what we aren't using?